_data/simple-icons.json

```diff
@@ -8420,6 +8420,11 @@
             "hex": "FD755C",
             "source": "https://kitsu.io"
         },
+        {
+            "title": "Kiwix",
+            "hex": "000000",
+            "source": "https://wiki.kiwix.org/wiki/Logo"
```

Comment:
```suggestion
            "source": "https://wiki.kiwix.org/wiki/Logo",
            "license": {
                "type": "CC-BY-SA-4.0"
            }
```

According to https://wiki.kiwix.org/wiki/Logo#/media/File:Kiwix_logo_v3.svg, we can add the following license:
![image](https://github.com/user-attachments/assets/210cd5a0-f0fb-4b2b-b679-3e2def0fdcbd)